setup.py

```diff
@@ -1,10 +1,11 @@
 import os
-from setuptools import setup
+from skbuild import setup
 from setuptools.dist import Distribution
 import sys
 
 NAME = 'pygmo'
-VERSION = '@pygmo_VERSION@'
```

Comment:
It's not ideal to have to remember to update the version number of pygmo in several different places when making new releases. I'd propose to have a simple txt file in the root directory of the project called ``pygmo_version`` containing just ``2.18.0`` (say) and then reading the version number from this file from both CMake and ``setup.py``.